### DISCUSSION TITLE
Is there a way to disable the vpn?

### DISCUSSION THREAD

**Discussion by rpgdev** [haugene/docker-transmission-openvpn#2816] [Q&A]
Would like to know if there's an option to start the container with the vpn disabled.

**Comment by edgd1er** [comment]
At the moment no. The container security (kill switch) is based on events from openvpn (tunnel up/down) to start and stop transmission. That feature would request a major refactoring, weakening the safety of those relying on safe vpn connection.
I m sure there are plenty of plain transmission images avaliable , why not use one of them ?

**Comment by rpgdev** [comment]
> At the moment no. The container security (kill switch) is based on events from openvpn (tunnel up/down) to start and stop transmission. That feature would request a major refactoring, weakening the safety of those relying on safe vpn connection. I m sure there are plenty of plain transmission images avaliable , why not use one of them ?

Thought it would be helpful for debugging purposes.. guess a different image can be used for the same purpose I guess.
